vertx-web-api-contract/src/test/java/io/vertx/ext/web/contract/openapi3/OpenAPI3RouterFactoryTest.java

```diff
@@ -73,7 +75,60 @@ public void tearDown() throws Exception {
   }
 
   @Test
-  public void loadPetStoreAndTestSomething() throws Exception {
+  public void loadPetStoreFromContentAndTestSomething() throws Exception {
+    CountDownLatch latch = new CountDownLatch(1);
+    final Router[] router = {null};
```

Comment:
@ahanin perhaps you could extract the setup code into a common function and reduce code duplication between yours & `loadPetStoreFromFileAndTestSomething` test, thoughts?

Reply:
@azagniotov Split it into two instead, a router builder and a test itself. Also had to move some test files to their respective package directories.